code/modules/client/preference_setup/loadout/loadout_fluffitems_vr.dm

```diff
@@ -337,12 +337,15 @@
 	ckeywhitelist = list("jemli")
 	character_name = list("Jemli")
 
+//TFF 28/11/19 - Expired permit removal
+/*
 /datum/gear/fluff/jeremiah_permit
 	path = /obj/item/clothing/accessory/permit/gun/fluff/ace
 	display_name = "Ace's Gun Permit"
 	ckeywhitelist = list("jertheace")
 	character_name = list("Jeremiah Acacius")
 	allowed_roles = list("Colony Director", "Warden", "Head of Security")
+*/
 
 /datum/gear/fluff/jeremiah_gun
```

Comment:
Can to comment this out too?

Reply:
Gun, holster, and ammo in one fell swoop?

Reply:
Gonna assume it's just for the gun. Commented gun out. Ammo, holster, combat boots, and combat uniform are still coded in for your loadout, however.

Reply:
Oh, I figured you would have removed the ammo, too.

Reply:
I commented the ammo out, but left the holster since, well, the jobs you get that for have access to guns anyway so you might want to keep that holster.